### PR TITLE
Update AutoML-NLP.md

### PR DESCRIPTION
**Pull request by liususan091219**

### CODE REVIEW

File: website/docs/Examples/AutoML-NLP.md [hunk 1]
```diff
@@ -9,25 +9,29 @@ pip install "flaml[nlp]"
...
 ### A simple sequence classification example
 
+To use the sequence classification module in FLAML, simply load your data, convert it to the pandas format, and call automl.fit:
+
 ```python
 from flaml import AutoML
 from datasets import load_dataset
 
-train_dataset = load_dataset("glue", "mrpc", split="train").to_pandas()
+train_dataset = load_dataset("glue", "mrpc", split="train").to_pandas()
```
Comment (sonichi) [inline website/docs/Examples/AutoML-NLP.md:18]:
```suggestion
train_dataset = load_dataset("glue", "mrpc", split="train").to_pandas()
```

File: website/docs/Examples/AutoML-NLP.md [hunk 2]
```diff
@@ -63,19 +69,21 @@ train_dataset = (
...
 X_val = dev_dataset[custom_sent_keys]
 y_val = dev_dataset[label_key]
 
 automl = AutoML()
 automl_settings = {
-    "gpu_per_trial": 0,
-    "time_budget": 20,
-    "task": "seq-regression",
-    "metric": "rmse",
+    "gpu_per_trial": 0,
```
Comment (sonichi) [inline website/docs/Examples/AutoML-NLP.md:83]:
```suggestion
    "gpu_per_trial": 0,
```

File: website/docs/Examples/AutoML-NLP.md [hunk 3]
```diff
@@ -137,72 +138,126 @@ automl.fit(
...
+y_val = dev_dataset[label_key]
+
+automl = AutoML()
+automl_settings = {
+        "gpu_per_trial": 0,
+        "max_iter": 2,
+        "time_budget": 5,                       # specify the time budget 
+        "task": "token-classification",         # specify the task to token classification
+        "metric": "seqeval",                    # specify the metric
+    }
```
Comment (sonichi) [inline website/docs/Examples/AutoML-NLP.md:173]: fix indent

File: website/docs/Examples/AutoML-NLP.md [hunk 4]
```diff
@@ -137,72 +138,126 @@ automl.fit(
...
+
+automl_settings["custom_hpo_args"] = {
+        "model_path": "bert-base-uncased",
+        "output_dir": "test/data/output/",
+        "ckpt_per_epoch": 1,
+        "fp16": False,
+    }
```
Comment (sonichi) [inline website/docs/Examples/AutoML-NLP.md:180]: fix indentation

File: website/docs/Examples/AutoML-NLP.md [hunk 5]
```diff
@@ -137,72 +138,126 @@ automl.fit(
...
+
+automl = AutoML()
+automl_settings = {
+        "gpu_per_trial": 0,
+        "max_iter": 2,
+        "time_budget": 5,                      # specify the time budget 
+        "task": "multichoice-classification",  # specify the task to multiple choice
+        "metric": "accuracy",                  # specify the evaluation metric
+        "log_file_name": "seqclass.log",
     }
```
Comment (sonichi) [inline website/docs/Examples/AutoML-NLP.md:237]: fix indentation

File: website/docs/Examples/AutoML-NLP.md [hunk 6]
```diff
@@ -137,72 +138,126 @@ automl.fit(
...
-  "use_cache": true,
-  "vocab_size": 32128
-}
+
+automl_settings["custom_hpo_args"] = {
+        "model_path": "google/electra-small-discriminator",
+        "output_dir": "test/data/output/",
+        "ckpt_per_epoch": 1,
+        "fp16": False,
+    }
```
Comment (sonichi) [inline website/docs/Examples/AutoML-NLP.md:244]: fix indentation

**Review by sonichi — COMMENTED** [review]
It's hard to see the schema of the data in the current examples. What about
1. add some concrete data examples in comments
2. for at least one task, make one example which does not use hf datasets

File: website/docs/Examples/AutoML-NLP.md [hunk 7]
```diff
@@ -9,25 +9,29 @@ pip install "flaml[nlp]"
 
 ### A simple sequence classification example
 
+To use the sequence classification module in FLAML, simply load your data, convert it to the pandas format, and call automl.fit:
```
Comment (sonichi) [inline website/docs/Examples/AutoML-NLP.md:12]:
```suggestion
To perform the sequence classification task in FLAML, simply load your data, convert it to the pandas format, and use `AutoML.fit()`:
```

File: website/docs/Examples/AutoML-NLP.md [hunk 8]
```diff
@@ -137,72 +138,126 @@ automl.fit(
...
+y_train = train_dataset[label_key]
+
+X_val = dev_dataset[custom_sent_keys]
+y_val = dev_dataset[label_key]
+
+automl = AutoML()
+automl_settings = {
+    "gpu_per_trial": 0,
+    "max_iter": 2,
+    "time_budget": 5,                       # specify the time budget
```
Comment (sonichi) [inline website/docs/Examples/AutoML-NLP.md:170]:
```suggestion
    "time_budget": 50,                       # specify the time budget 
```

File: website/docs/Examples/AutoML-NLP.md [hunk 9]
```diff
@@ -137,72 +138,126 @@ automl.fit(
...
+y_train = train_dataset[label_key]
+
+X_val = dev_dataset[custom_sent_keys]
+y_val = dev_dataset[label_key]
+
+automl = AutoML()
+automl_settings = {
+    "gpu_per_trial": 0,
+    "max_iter": 2,
+    "time_budget": 5,                      # specify the time budget
```
Comment (sonichi) [inline website/docs/Examples/AutoML-NLP.md:233]:
```suggestion
    "time_budget": 50,                      # specify the time budget 
```

**Review by sonichi — CHANGES_REQUESTED** [review]
My previous comments haven't been addressed.

**Comment by liususan091219** [comment]
> My previous comments haven't been addressed.

addressed

**Comment by sonichi** [comment]
> > My previous comments haven't been addressed.
> 
> addressed

I still can't tell the data schema. Please include that information in at least one example.

File: website/docs/Examples/AutoML-NLP.md [hunk 10]
```diff
@@ -9,25 +9,35 @@ pip install "flaml[nlp]"
 
 ### A simple sequence classification example
 
+An example of sequence classification data is the Stanford Sentiment Treebank data (SST) data:
+
+sentence  label 
+hide new secretions from the parental units       0 
+contains no wit , only labored gags       0
```
Comment (sonichi) [inline website/docs/Examples/AutoML-NLP.md:16]: include one positive example.
use markdown to format it so that it's clear to see it's a table.

**Review by sonichi — COMMENTED** [review]
merge with main
